website/source/docs/auth/kerberos.html.md

```diff
@@ -0,0 +1,225 @@
+authentication in Active Directory, and its purpose is to _distribute_
+a network's authentication workload.
+
+Vault's Kerberos auth method was originally written by the folks at 
+[Winton](https://github.com/wintoncode), to whom we owe a special thanks
+for both originally building the plugin, and for collaborating to bring 
+it into HashiCorp's maintenance.
+
+## Prerequisites
+
+Kerberos is a very hands-on auth method. Other auth methods like 
+[LDAP](https://www.vaultproject.io/docs/auth/ldap.html) and 
+[Azure](https://www.vaultproject.io/docs/auth/azure.html) only require
+a cursory amount of knowledge for configuration and use.
+Kerberos, on the other hand, is best used by people already familiar
+with it. We recommend that you use simpler authentication methods if
+your use case is achievable through them. If not, we recommend that 
+before approaching Kerberos, you become familiar with its fundamentals.
+
+- [MicroNugget: How Kerberos Works in Windows Active Directory](https://www.youtube.com/watch?v=kp5d8Yv3-0c)
+- [MIT's Kerberos Documentation](https://web.mit.edu/kerberos/)
+- [Kerberos: The Definitive Guide](https://www.amazon.com/Kerberos-Definitive-Guide-ebook-dp-B004P1J81C/dp/B004P1J81C/ref=mt_kindle?_encoding=UTF8&me=&qid=1573685442)
+
+Regardless of how you gain your knowledge, before using this auth method,
+ensure you are comfortable with Kerberos' high-level architecture, and
+ensure you've gone through the exercise of:
+
+- Creating a valid `krb5.conf` file
+- Creating a valid `keytab` file
+- Authenticating to your domain server with your `keytab` file using `kinit`
+
+With that knowledge in hand, and with an environment that's already tested
+and confirmed working, you will be ready to use Kerberos with Vault.
+
+## Configuration
+
+* Enable Kerberos authentication in Vault:
+
+```text
+$ vault auth enable \
+    -passthrough-request-headers=Authorization \
+    -allowed-response-headers=www-authenticate \
+    kerberos
+```
+   
+* Create a `keytab` for the Kerberos plugin:
+
+```text
+$ ktutil
+ktutil:  addent -password -p your_service_account@REALM.COM -e aes256-cts -k 1
+Password for your_service_account@REALM.COM:
+ktutil:  list -e
+slot KVNO Principal
+---- ---- ---------------------------------------------------------------------
+  1    1            your_service_account@REALM.COM (aes256-cts-hmac-sha1-96)
+ktutil:  wkt vault.keytab
+```
+   
+   The KVNO (`-k 1`) should match the KVNO of the service account. An error will show in the Vault logs if this is incorrect.
+   
+   Different encryption types can also be added to the `keytab`, for example `-e rc4-hmac` with additional `addent` commands.
+   
+   Then base64 encode it:
+   
+```text
+$ base64 vault.keytab > vault.keytab.base64
+```
+
+* Configure the Kerberos auth method with the `keytab` and 
+entry name that will be used to verify inbound login 
+requests:
+
+```text
+$ vault write auth/kerberos/config \
+    keytab=@vault_svc.keytab.base64 \
```

Comment:
nit: the filename here doesn't match the previous step.

Reply:
Good eye! Fixing.